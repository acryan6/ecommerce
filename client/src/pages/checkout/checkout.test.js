import React from "react";
import { shallow } from "enzyme";

import { CheckoutPage } from "./checkout.component";
import CheckoutItem from "../../components/checkout-item/checkout-item.component";

describe("CheckoutPage", () => {
  const mockProps = {
    cartItems: [
      { id: 1, name: "blue hat", imageUrl: "hat.com", price: 500, quantity: 1 },
    ],
    total: 500,
  };

  const wrapper = shallow(<CheckoutPage {...mockProps} />);

  it("should render page", () => {
    expect(wrapper).toMatchSnapshot();
  });

  it("should render CheckoutItem if cart is not empty", () => {
    expect(wrapper.exists(CheckoutItem)).toBe(true);
  });
});
