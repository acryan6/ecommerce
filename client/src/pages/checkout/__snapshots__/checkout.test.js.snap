// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CheckoutPage should render page 1`] = `ShallowWrapper {}`;
